test/unit/vpc.test.ts

```diff
@@ -0,0 +1,25 @@
+import { get } from "lodash";
+import { pluginConfigExt, runServerless } from "../utils/runServerless";
+
+type CfTemplate = {
+    Resources: Record<string, unknown>;
+    Outputs?: Record<string, unknown>;
+};
+
+function expectUserDynamoStatementIsAdded(cfTemplate: CfTemplate) {
```

Comment:
Is "Dynamo" a leftover from another branch?

Reply:
yep, definitly copy/pasted some test code and did not cleaned-it-up :sweat_smile: